File: sanghyun/programmers-2/예상 대진표.java
class Solution
{
    public int solution(int n, int a, int b)
    {
        int answer = 0;
        while (true) {
            answer++;
            a = (a+1) / 2;
            b = (b+1) / 2;
            if (a == 1 && b == 1 || a == b){
                return answer;
            }
        }
    }
}

File: sanghoon/programmers-2/예상 대진표.py
def solution(n,a,b):
    answer = 0
    while True :
        answer += 1

        a = (a + 1) // 2
        b = (b + 1) // 2

        if a == b :
            break
    print(answer)

n = 8
a = 4
b = 7
solution(n,a,b)